recipes/modulegraph2/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "modulegraph2" %}
+{% set version = "2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/modulegraph2-{{ version }}.tar.gz
+  sha256: 153d2a7dd0eaaffcf608b3890f6cda3f7ac69d91d309f8eed7313269adfb3de2
+
+build:
+  number: 0
+  skip: true   # [py>=40 or py2k]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
+    - flit
+  run:
+    - dataclasses  # [py<37]
```

Comment:
This could be enabled for all python versions to allow the package to be noarch: Python?

Reply:
@chrisburr I have removed this. I didn't know we had dataclasses package for py>=37. Could you review again? Thanks!